README.md

```diff
@@ -1,14 +1,29 @@
-NYPLBase
+#NYPLBase#
 ========
-This README is a work in progress...
+NYPLBase (or familiarly, "Base") provides a "base" for all NYPL web properties. Color palette, typeface, default styles, and a bit of structure is all built into this code package. (For those familiar with Twitter Bootstrap, it's like the Library's own Bootstrap but with fewer options.) 
 
+The idea is that anyone who's building a web project for the Library, whether one of our internal dev teams, or an external consultant group, can download and use this code as the style base, and not have to worry about getting our brand right--it's already in there. Even if you're not using the code, you can still look at the demo page as a style guide.
 
-NYPLBase (or familiarly, "Base") provides a "base" for all NYPL web properties. Color palette, typeface, default styles, and a bit of structure is all built into this code package. (For those familiar with Twitter Bootstrap, it's like the Library's own Bootstrap but with fewer options.) 
+At this moment, May 2018, only three applications are still using NYPLBase. They are Location-App (previous Location-prototype), Research Collections, and Staff Profiles. Along with some old drupal pages. The applications after these are using NYPL Design Toolkit.
+
+##Usage##
+To update the styles for the apps that use NYPLBase, simply make a feature branch and make the changes. Then you need to use SASS to compile the changes to `min` files.
+
+Please run
+`sass css/style.scss:css/nyplbase.min.css --style compressed`. It should generate the min CSS and min CSS map under the name `nyplbase`.
+
+Please go to the particular project's documentation for more details about testing the styles.
+
+Location App - https://github.com/NYPL/locations-app
+
+##Deployment##
```

Comment:
This is open sourced, but this is specific to nypl. So maybe write `##NYPL Deployment Process##`, because not everyone should be able to access jenkins and `ux-static` is a private repo.